homeassistant/components/google_assistant/__init__.py

```diff
@@ -72,13 +68,13 @@ def _check_report_state(data):
             vol.Optional(
                 CONF_EXPOSED_DOMAINS, default=DEFAULT_EXPOSED_DOMAINS
             ): cv.ensure_list,
-            vol.Optional(CONF_API_KEY): cv.string,
             vol.Optional(CONF_ENTITY_CONFIG): {cv.entity_id: ENTITY_SCHEMA},
-            vol.Optional(CONF_ALLOW_UNLOCK): cv.boolean,
             # str on purpose, makes sure it is configured correctly.
             vol.Optional(CONF_SECURE_DEVICES_PIN): str,
             vol.Optional(CONF_REPORT_STATE, default=False): cv.boolean,
             vol.Optional(CONF_SERVICE_ACCOUNT): GOOGLE_SERVICE_ACCOUNT,
+            vol.Remove("allow_unlock"): cv.boolean,
+            vol.Remove("api_key"): cv.string,
```

Comment:
Please keep the constants (you can move them to the same file). I know it seem pointless. But it matches how we use constants..